IntelliTect.TestTools.Selenate/IntelliTect.TestTools.Selenate/IntelliTect.TestTools.Selenate.csproj

```diff
@@ -2,20 +2,19 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
+    <LangVersion>9.0</LangVersion>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <WarningLevel>4</WarningLevel>
+  </PropertyGroup>
 
-    <GenerateDocumentationFile>true</GenerateDocumentationFile>
-
-    <Version>1.3.0</Version>
-
-    <PackageReleaseNotes>Adding override to Browser.TakeScreenshot to hand in your own preferred path</PackageReleaseNotes>
-
-    <GeneratePackageOnBuild>false</GeneratePackageOnBuild>
-
-    <AssemblyVersion>1.3.0.0</AssemblyVersion>
-
-    <FileVersion>1.3.0.0</FileVersion>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <NoWarn>1701;1702;CA1303</NoWarn>
   </PropertyGroup>
 
+  <ItemGroup>
+    <None Remove="HandlerBase.cs~RF6d5b2d80.TMP" />
```

Comment:
This looks like a temp file. Is this needed?

Reply:
Removed